cmake/features.cmake

```diff
@@ -49,4 +49,4 @@ ie_dependent_option (ENABLE_PROFILING_ITT "ITT tracing of IE and plugins interna
 # Documentation build
 ie_option (ENABLE_DOCS "build docs using Doxygen" OFF)
 
-ie_dependent_option (ENABLE_FASTER_BUILD "Enable build features (PCH, UNITY) to speed up build time" OFF "CMAKE_VERSION VERSION_GREATER_EQUAL 3.16" OFF)
+ie_option (ENABLE_FASTER_BUILD "Enable build features (PCH, UNITY) to speed up build time" ON)
```

Comment:
Minimal supported CMake version is `3.13`, so we have to check the actual CMake version.

Reply:
we have minimum required version 3.13 in root cmake.

Reply:
Yes, generic requirement is `3.13`, but Unity and PCH requires `3.16`. Either we need to change base version check or leave this option as dependent.